runtime/component/dnd-mode.js

```diff
@@ -0,0 +1,634 @@
+'use strict'
+
+var property = require('@yoda/property')
+var logger = require('logger')('dndmode')
+
+var FSMCode = {
+  Start: 0,
+  End: 254,
+  WaitAsync: 255,
+  CheckSwitchOn: 1,
+  CheckSwitchOff: 2,
+  CheckStatusOff: 3,
+  CheckStatusOn: 4,
+  DNDModeDisabled: 5,
+  CheckStatusOnX: 6,
+  CheckStatusOffX: 7,
+  CheckTimeSuccess: 8,
+  CheckTimeFailed: 9,
+  DNDModeDisabledX: 10,
+  CheckTimeSuccessX: 11,
+  CheckTimeFailedX: 12,
+  CheckActivityTrue: 13,
+  CheckActivityFalse: 14,
+  CheckActivityTrueX: 15,
+  CheckActivityFalseX: 16,
+  DNDModeEnabled: 17,
+  CheckAgain: 18
+}
```

Comment:
Could this be defined in JSON to speed up the parsing time?

Reply:
OK, but I don't know where the json file should be stored, is there any suggestion?

Reply:
How about /etc/yoda/dnd/config.json?

Reply:
Actually these are not configs, and the directory `/etc/` can be redefined with wrong data.